### ISSUE TITLE
Error when posting with an attachment

### ISSUE THREAD

**Issue by jeffdanforth** [ActiveForums/ActiveForums#434]
We are getting the error "The PathNameAlias specified for dependency knockout.js does not exist in the ClientDependencyPathCollection" when trying to create a post with an attachment in the Active Forums module. We are running DNN 9.1.1 and Active Forums 6.3.3.

Thanks for any help you can provide.

**Comment by artfuld** [comment]
Hey @jeffdanforth do you have Knockout available in your DNN-site?

Settings => Extensions, In the dropdown named Showing switch to JavaScript Libraries

**Comment by jeffdanforth** [comment]
Yes, although we have three. Versions 3.3.0, 2.2.1 and 3.1.0. Plus Knockout Mapping 2.4.1.
